### PR TITLE
Storybook > Typography

### PR DESCRIPTION
**Pull request by mckelvey**
This PR updates Storybook Design System section to include Typography matching that in the Figma with two exceptions: the addition of `StyledButtonText` and `StyledLinkText` for consistent/easy “action” typography.

### CODE REVIEW

**Comment by mckelvey** [comment]
[Storybook > Typography](https://trello.com/c/BEWpTDdJ/13-storybook-typography)

File: __stories__/Colors.stories.js [hunk 1]
```diff
@@ -2,12 +2,11 @@ import React from 'react'
 import styled from 'styled-components'
 
 import StoryGrid from './StoryGrid'
-import { H2 } from '../components/ui/Typography'
 import theme from '../util/theme'
 
-const SectionTitle = styled(H2)`
+const SectionTitle = styled.h2`
```
Comment (royriver23) [inline __stories__/Colors.stories.js:7]: Nice improvement!

File: __stories__/Typography.stories.js [hunk 2]
```diff
@@ -7,63 +8,74 @@ import {
...
+      </div>
+      <div>
+        <Body1>Body 1</Body1>
+      </div>
+      <div>
+        <Body2>Body 2</Body2>
+      </div>
+      <div>
+        <StyledButtonText
+          style={{ backgroundColor: '#4a4a4a', borderColor: '#4a4a4a' }}
```
Comment (royriver23) [inline __stories__/Typography.stories.js:50]: are we planning to use named colors?

Comment (royriver23) [reply __stories__/Typography.stories.js:50]: I guess this is just to show the UI example component in storybook, so, you can skip this.

File: util/theme.js [hunk 3]
```diff
@@ -44,6 +44,11 @@ const theme = {
     md: `left top/0.8rem ${offsetDotSvgUrl}`,
     lg: `left top/1rem ${offsetRoundedDotSvgUrl}`,
   },
+
+  fontStacks: {
+    chicago: 'ChicagoFLFRegular, sans-serif',
+    geneva: 'Geneva, Verdana',
```
Comment (royriver23) [inline util/theme.js:50]: oh nice, I see what you did there

**Review by royriver23 — APPROVED** [review]

File: __stories__/Typography.stories.js (hunk 2 above)
Comment (mckelvey) [reply __stories__/Typography.stories.js:50]: 👍 You guessed it. :)

**Review by edderrd — APPROVED** [review]

**Pull request merged**